CHANGELOG.md

```diff
@@ -5,7 +5,7 @@ Full documentation for rocPRIM is available at [https://rocm.docs.amd.com/projec
 ## rocPRIM 3.3.0 for ROCm 6.3.0
 
 ### Added
-
+* Updated the default value for the `-a` argument from `rmake.py` to `gfx906:xnack-,gfx1030,gfx1100,gfx1101,gfx1102,gfx1151,gfx1200,gfx1201`.
```

Comment:
From the code it looks like the default value has changed to `default_gpus` from `gfx906;gfx1030;gfx1100;gfx1101;gfx1102`

Can you let me know if this is right? Or clarify what has changed?

Reply:
Yes, but default_gpus is just a variable holding the string `gfx906:xnack-,gfx1030,gfx1100,gfx1101,gfx1102,gfx1151,gfx1200,gfx1201`

Reply:
```suggestion
* * Changed the default value of `rmake.py -a` to `default_gpus`. This is equivalent to `gfx906:xnack-,gfx1030,gfx1100,gfx1101,gfx1102,gfx1151,gfx1200,gfx1201`.
```